src/frontend/src/components/irregularity.ts

```diff
@@ -0,0 +1,48 @@
+import { html, TemplateResult } from "lit-html";
+import { warningIcon, closeIcon } from "./icons";
+
+interface irregularityProps {
+  message: string | TemplateResult;
+  additionalClasses?: string[];
+  type?: "warning" | "info" | "error";
```

Comment:
Looks like this isn't used, can we remove it until necessary?

Reply:
I think we should leave this one in, it will definitely give guidance to whoever wants to extend this to use it for notifications or positive feeback.